examples/drdonut-nextjs/utils/callFunctions.ts

```diff
@@ -1,5 +1,5 @@
 'use client';
-import { UltravoxSession, UltravoxSessionStatus, Transcript, UltravoxSessionStateChangeEvent, UltravoxSessionState } from 'ultravox-client';
+import { UltravoxSession, UltravoxSessionStatus, Transcript } from 'ultravox-client';
 let UVSession: UltravoxSession | null = null;
```

Comment:
Can we also rename this uvSession please?

Reply:
done